src/lib/Scenes/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/ArtistArtworkGrid.tsx

```diff
@@ -0,0 +1,54 @@
+import { ArtistArtworkGrid_artwork } from "__generated__/ArtistArtworkGrid_artwork.graphql"
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Header } from "../../Header"
+
+interface ArtistArtworkGridProps {
+  artwork: ArtistArtworkGrid_artwork
+}
+
+export class ArtistArtworkGrid extends React.Component<ArtistArtworkGridProps> {
+  render() {
+    const {
+      artwork: { artist },
+    } = this.props
+
+    if (!artist) {
+      return null
+    }
+    const artworks = artist.artworks_connection.edges.map(({ node }) => node)
+
+    return (
+      <>
+        <Header title={`Other works by ${artist.name}`} />
+        <GenericGrid artworks={artworks} />
+      </>
+    )
+  }
+}
+
+export const ArtistArtworkGridFragmentContainer = createFragmentContainer(ArtistArtworkGrid, {
+  artwork: graphql`
+    fragment ArtistArtworkGrid_artwork on Artwork {
+      id
+      artist {
+        name
+        artworks_connection(first: 8, sort: PUBLISHED_AT_DESC, exclude: $excludeArtworkIds)
+          @connection(key: "GenericGrid_artworks_connection") {
+          pageInfo {
```

Comment:
looks like `pageInfo` is not being used. Also you don't need the @connection directive since this is not being used in a pagination container